formulas/excel.py

```diff
@@ -67,6 +72,67 @@ def load(self, filename):
         self.pushes(*book.worksheets, context=context)
         return self
 
+
+    def _load_bin(self, fn):
+        with open(fn, 'rb') as f:
+            return dill.load(f)
+
+    def _save_bin(self, obj, fn):
+        with open(fn, 'wb') as f:
+            dill.dump(obj, f)
+
+    def save_bin(self, fn):
+        self._save_bin(self, fn + ".fxl")
+
+    def _update_constant_value(self, attr, value):
```

Comment:
The reason for all of this setting is because tokens (`START`, `EMPTY`, etc.) compare uses `is` which checks if the token is the actual memory address. When loading binary file the memory location of the tokens is different.

*THIS IS NOT MEANT TO BE COMMITTED* - I am posting this PR to figure out the best way to handle it. The worksheet I have takes over 5 hours to compile so I really want a way be able to save off the excel model and reload it.